### PR TITLE
Change name of eRequesting ServiceRequest to eRequesting Diagnostic Request (FHIR-46842)

### PR DESCRIPTION
**Pull request by JayMurdoch**
Fix for [FHIR-46842](https://jira.hl7.org/browse/FHIR-46842)

* Changed profile name, id, title, file name of eRequesting ServiceRequest to eRequesting Diagnostic Request.
* Changed references to parent profile in eRequesting Imaging and Pathology Request profiles.
* Change log entry.

### CODE REVIEW

File: input/fsh/au-erequesting-diagnosticrequest.fsh [hunk 1]
```diff
@@ -1,7 +1,7 @@
-Profile: AUeRequestingServiceRequest
+Profile: AUeRequestingDiagnosticRequest
```
Comment (dbojicic) [inline input/fsh/au-erequesting-diagnosticrequest.fsh:1]: Commenting here to get an inline comment. 
Please update the example in ['Example: AU eRequesting ServiceRequest profile showing identifier labelled Must Support' in the Conformance page. ](https://build.fhir.org/ig/hl7au/au-fhir-erequesting/branches/master/conformance.html#presentation-of-elements-labelled-must-support-in-raw-representations)

File: input/fsh/au-erequesting-diagnosticrequest.fsh [hunk 2]
```diff
@@ -1,7 +1,7 @@
...
 Parent: AUBaseDiagnosticRequest
-Id: au-erequesting-servicerequest
-Title: "AU eRequesting ServiceRequest"
```
Comment (dbojicic) [inline input/fsh/au-erequesting-diagnosticrequest.fsh:4]: Commenting here to get an inline comment. 
Please update theAUeREqDI mappings into AUeRequesting table: https://build.fhir.org/ig/hl7au/au-fhir-erequesting/branches/master/auereqdi.html#auereqdi-mappings-into-au-erequesting

File: input/fsh/au-erequesting-diagnosticrequest.fsh [hunk 3]
```diff
@@ -1,7 +1,7 @@
...
+Id: au-erequesting-diagnosticrequest
```
Comment (dbojicic) [inline input/fsh/au-erequesting-diagnosticrequest.fsh:3]: Commenting here to get an inline comment. 

Please update AU Variance Statement (link and title).

File: input/pagecontent/changes.md [hunk 4]
```diff
@@ -9,3 +9,5 @@ This change log documents the significant updates and resolutions implemented fr
 - New profiles: 
   - AU eRequesting Task ([FHIR-46947](https://jira.hl7.org/browse/FHIR-46947), [FHIR-46950](https://jira.hl7.org/browse/FHIR-46950), [FHIR-46956](https://jira.hl7.org/browse/FHIR-46956), [FHIR-46958](https://jira.hl7.org/browse/FHIR-46958), [FHIR-46959](https://jira.hl7.org/browse/FHIR-46959), [FHIR-47081](https://jira.hl7.org/browse/FHIR-47081), [FHIR-47082](https://jira.hl7.org/browse/FHIR-47082), [FHIR-47083](https://jira.hl7.org/browse/FHIR-47083), [FHIR-47084](https://jira.hl7.org/browse/FHIR-47084), [FHIR-47086](https://jira.hl7.org/browse/FHIR-47086), [FHIR-47102](https://jira.hl7.org/browse/FHIR-47102).
   - AU eRequesting Coverage ([FHIR-46848](https://jira.hl7.org/browse/FHIR-46848))083](https://jira.hl7.org/browse/FHIR-47083), [FHIR-47084](https://jira.hl7.org/browse/FHIR-47084), [FHIR-47086](https://jira.hl7.org/browse/FHIR-47086), [FHIR-47102](https://jira.hl7.org/browse/FHIR-47102).
+- Made the following changes to AU eRequesting Diagnostic Request:
+  - changed name of profile from AU eRequesting ServiceRequest to AU eRequesting Diagnostic Request [FHIR-46842](https://jira.hl7.org/browse/FHIR-46842)
```
Comment (dbojicic) [inline input/pagecontent/changes.md:13]: This implies StructureDefinition.name change where we changed more. Maybe say something like: Renamed <AU eRequesting ServiceRequest> to be <AU eRequesting Diagnostic Request>

File: input/fsh/au-erequesting-diagnosticrequest.fsh [hunk 5]
```diff
@@ -1,7 +1,7 @@
...
+Title: "AU eRequesting Diagnostic Request"
```
Comment (dbojicic) [inline input/fsh/au-erequesting-diagnosticrequest.fsh:4]: Commenting here to get an inline comment. 

Please update CapabilityStatements - all capability statements and the source file for the Summary of Must Support and References Between Profiles will need to be updated.

File: input/fsh/au-erequesting-diagnosticrequest.fsh (hunk 1 above)
Comment (dbojicic) [inline input/fsh/au-erequesting-diagnosticrequest.fsh:1]: Commenting here to get an inline comment. 

Please update profile title and link in the Terminology page.

File: input/fsh/au-erequesting-servicerequest-path.fsh [hunk 6]
```diff
@@ -1,5 +1,5 @@
 Profile: AUeRequestingPathologyRequest
-Parent: AUeRequestingServiceRequest
+Parent: AUeRequestingDiagnosticRequest
 Id: au-erequesting-servicerequest-path
 Title: "AU eRequesting Pathology Request"
 Description: "This profile sets minimum expectations for a ServiceRequest resource that is used to record, search, and fetch information about pathology requests for a patient. It is based on the [AU eRequesting ServiceRequest](https://build.fhir.org/ig/hl7au/au-fhir-erequesting/StructureDefinition-au-erequesting-servicerequest.html) and identifies the additional constraints, extensions, vocabularies and value sets that **SHALL** be present in the ServiceRequest when conforming to this profile."
```
Comment (dbojicic) [inline input/fsh/au-erequesting-servicerequest-path.fsh:5]: Need to update title and link here as well.

File: input/fsh/au-erequesting-servicerequest-imag.fsh [hunk 7]
```diff
@@ -1,5 +1,5 @@
 Profile: AUeRequestingImagingRequest
-Parent: AUeRequestingServiceRequest
+Parent: AUeRequestingDiagnosticRequest
 Id: au-erequesting-servicerequest-imag
 Title: "AU eRequesting Imaging Request"
 Description: "This profile sets minimum expectations for a ServiceRequest resource that is used to record, search, and fetch information about diagnostic imaging requests for a patient. It is based on the [AU eRequesting ServiceRequest](https://build.fhir.org/ig/hl7au/au-fhir-erequesting/StructureDefinition-au-erequesting-servicerequest.html) and identifies the additional constraints, extensions, vocabularies and value sets that **SHALL** be present in the ServiceRequest when conforming to this profile."
```
Comment (dbojicic) [inline input/fsh/au-erequesting-servicerequest-imag.fsh:5]: Need to update title and link here as well.

**Review by dbojicic — CHANGES_REQUESTED** [review]
Please see comments.

File: input/fsh/au-erequesting-diagnosticrequest.fsh (hunk 1 above)
Comment (JayMurdoch) [reply input/fsh/au-erequesting-diagnosticrequest.fsh:1]: Amended, thanks.

Comment (JayMurdoch) [reply input/fsh/au-erequesting-diagnosticrequest.fsh:1]: Amended, thanks.

File: input/fsh/au-erequesting-diagnosticrequest.fsh (hunk 2 above)
Comment (JayMurdoch) [reply input/fsh/au-erequesting-diagnosticrequest.fsh:4]: Amended, thanks.

File: input/fsh/au-erequesting-diagnosticrequest.fsh (hunk 3 above)
Comment (JayMurdoch) [reply input/fsh/au-erequesting-diagnosticrequest.fsh:3]: Amended, thanks.

File: input/fsh/au-erequesting-diagnosticrequest.fsh (hunk 5 above)
Comment (JayMurdoch) [reply input/fsh/au-erequesting-diagnosticrequest.fsh:4]: Updated CapStats + resourcereferencesummary file.

File: input/fsh/au-erequesting-servicerequest-imag.fsh (hunk 7 above)
Comment (JayMurdoch) [reply input/fsh/au-erequesting-servicerequest-imag.fsh:5]: Amended, thanks.

File: input/fsh/au-erequesting-servicerequest-path.fsh (hunk 6 above)
Comment (JayMurdoch) [reply input/fsh/au-erequesting-servicerequest-path.fsh:5]: Amended, thanks.

File: input/fsh/au-erequesting-diagnosticrequest.fsh (hunk 1 above)
Comment (dbojicic) [reply input/fsh/au-erequesting-diagnosticrequest.fsh:1]: Please updated profile title in the example caption as well.

**Review by dbojicic — CHANGES_REQUESTED** [review]
Small change requested.

File: input/fsh/au-erequesting-diagnosticrequest.fsh (hunk 1 above)
Comment (JayMurdoch) [reply input/fsh/au-erequesting-diagnosticrequest.fsh:1]: Amended, thanks.

**Review by dbojicic — APPROVED** [review]

**Review by michael-wilson-au — APPROVED** [review]

**Pull request merged**